### ISSUE TITLE
About test_error_count

### ISSUE THREAD

**Issue by wangdahao** [NVIDIA-AI-IOT/jetbot#154]
Hi!
There seems to be a problem with the calculation of "test_error_count".
For example,when the value  of expression "torch.abs(labels - outputs.argmax(1))" is "tensor([0, 2, 0, 1])".It jsut mean there have 2images test error,not 3.
So I think "torch.sum(torch.abs(labels - outputs.argmax(1))))" should be instead of "numpy.count_nonzero(torch.abs(labels - outputs.argmax(1))))". 
One more thing,for two labels classifiction,test_error_count is equal to value of expression "torch.sum(torch.abs(labels - outputs.argmax(1))))" by lucky coincidence. When faced with thress classifiction that is wrong.
Thank you.
